semantic_router/index/base.py

```diff
@@ -109,7 +110,11 @@ def delete_index(self):
         raise NotImplementedError("This method should be implemented by subclasses.")
 
     def _sync_index(
-        self, local_route_names: List[str], local_utterances: List[str], dimensions: int
+        self,
+        local_route_names: List[str],
+        local_utterances: List[str],
+        dimensions: int,
+        local_function_schemas: List[str] = None,  # type: ignore
```

Comment:
can we remove `# type: ignore` here and confirm it works as expected?

```suggestion
        local_function_schemas: List[str] | None = None,
```